src/cudaMain.cu

```diff
@@ -1338,8 +1338,8 @@ int cudaMainStreams(const UINT *voxel,
             START_TIMER(TIMERS::IMAGE_ROTATION)
           }
 #endif
-          const double alpha = cos(-Eangle);
-          const double beta = sin(-Eangle);
+          const double alpha = cos(Eangle);
+          const double beta = sin(Eangle);
```

Comment:
How have we tested that this sign flip is correct?

Reply:
This was a bug that somehow git missed out. The current one is correct. The negative sign should not be there.

Reply:
Yes, tested to be correct and match algorithm 1.

Reply:
I am adding a followup issue for this as #10 so we don't lose the thread of adding a unit test/functional test.